src/main/java/school/lemon/changerequest/java/employees/Accountant.java

```diff
@@ -0,0 +1,31 @@
+package school.lemon.changerequest.java.employees;
+
+public class Accountant extends Employee {
```

Comment:
I'd rather have some additional interfaces for this hierarchy, but in general the hierarchy is OK.